src/braket/circuits/gate.py

```diff
@@ -28,15 +28,13 @@
 
 
 class Gate(QuantumOperator):
-    """
-    Class `Gate` represents a quantum gate that operates on N qubits. Gates are considered the
+    """Class `Gate` represents a quantum gate that operates on N qubits. Gates are considered the
     building blocks of quantum circuits. This class is considered the gate definition containing
     the metadata that defines what a gate is and what it does.
     """
 
     def __init__(self, qubit_count: Optional[int], ascii_symbols: Sequence[str]):
-        """
-        Args:
+        """Args:
```

Comment:
Does this need an summary? You're adding them to other `__init__` methods

Reply:
Fixed